### ISSUE TITLE
Sync finish-args with the KDE Apps Beta repo

### ISSUE THREAD

**Issue by herzenschein** [flathub/org.kde.kate#32]
When I initialize or clone any git repository and open its files with Kate, the Project pane on the left provided by the Projects plugin doesn't update to show the contents of the repository.

Additionally, the Git pane, also provided by the Projects plugin, doesn't show any changes if I edit and save any files in the repository, even when I click the Refresh button.

**Comment by Erick555** [comment]
Does projects plugin use git? There us no git available in app flatpak environment.

**Comment by travier** [comment]
We could base Kate directly on the Sdk to get basic tools in but in general, Kate should run commands outside the Flatpak. Wondering why this is not happening here.

**Comment by herzenschein** [comment]
Ah, I just realized I was testing the kdeapps Kate rather than flathub-beta Kate. That was my bad.

I noticed some disparity between the flathub-beta and kdeapps versions though. It seems the kdeapps Kate is missing the following session bus permissions that exist in the flathub-beta Kate:

* org.freedesktop.Flatpak (Session Bus) <-- This one makes the Projects plugin work
* org.freedesktop.UDisks2 (System Bus) <-- No idea what this one is for

I also noticed that the lack of `org.freedesktop.Flatpak` makes opening the embedded terminal simply terminate Kate (not even crash).
